packages/@aws-cdk/aws-servicecatalog/package.json

```diff
@@ -75,24 +75,35 @@
   "devDependencies": {
     "@types/jest": "^26.0.23",
     "cdk-build-tools": "0.0.0",
+    "cdk-integ-tools": "0.0.0",
     "cfn2ts": "0.0.0",
     "pkglint": "0.0.0",
     "@aws-cdk/assert-internal": "0.0.0"
   },
   "dependencies": {
+    "@aws-cdk/aws-iam": "0.0.0",
+    "@aws-cdk/cloud-assembly-schema": "0.0.0",
```

Comment:
What do we need `"@aws-cdk/cloud-assembly-schema` for?

Reply:
removed, was used in our prev PR